app/jenkins-for-jira-ui/src/components/icons/NoServersConnectedIcon.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { cx } from '@emotion/css';
+
+type NoServersConnectedIconProps = {
+	containerClassName: string
+};
+
+export function NoServersConnectedIcon({ containerClassName }: NoServersConnectedIconProps): JSX.Element {
+	return (
+		<div className={cx(containerClassName)} data-testid="noServersConnectedIcon">
+			<svg width="160" height="160" viewBox="0 0 160 160" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I feel like I asked this last time and have completely forgot, but why do we need to put in the svg like this raw instead og a img tag?